docs/unifiedpush/aerogear-push-android/google-setup.asciidoc

```diff
@@ -1,45 +1,40 @@
 [[google-setup]]
-== Google Related Setup
+== Obtaining Firebase Cloud Messaging Credentials
 
-Before the Android application is able to receive the notifications, you must authenticate yourself with Google as a developer. Luckily this is both easy and free to do, and doesn't require much set-up. Like iOS, you are only able to test out your push notifications in a real device.
+Before the Android application is able to receive the notifications, you must set up access to the Firebase Console. Luckily, this is both easy and free to do, and doesn't require much set-up. As with iOS, you are only able to test out your push notifications in a real device. This section shows you how to obtain the credentials necessary to set up Firebase Cloud Messaging for your app, including:
```

Comment:
I'd say you must set up access either "to Firebase" or "to Firebase Cloud Messaging", instead of "to the Firebase Console". 

IIUC, Firebase Console is only the web UI for controlling Firebase. Therefore, I'd say the situation is, that you _use the Firebase Console_ to set up _access to Firebase Cloud Messaging_.